cluster/cluster.go

```diff
@@ -28,6 +30,8 @@ var (
 	Mode    ModeType
 	Manager *ClusterManager
 	cfg     *memberlist.Config
+
+	InsufficientShardsAvailable = NewError(http.StatusServiceUnavailable, fmt.Errorf("Insufficient shards available."))
```

Comment:
fmt.Errorf is good when you need to format things. If you don't, can just use `errors.New()` instead.